bin/install-php-phpunit.sh

```diff
@@ -0,0 +1,128 @@
+    git clone --depth=1 --branch=1.1.1 git://github.com/sebastianbergmann/php-invoker.git
+    git clone --depth=1 --branch=1.1.2 git://github.com/sebastianbergmann/php-text-template.git
+    git clone --depth=1 --branch=1.0.3 git://github.com/sebastianbergmann/php-timer.git
+    git clone --depth=1 --branch=1.1.4 git://github.com/sebastianbergmann/php-token-stream.git
+    git clone --depth=1 --branch=1.1   git://github.com/sebastianbergmann/phpunit-mock-objects.git
+    git clone --depth=1 --branch=1.1   git://github.com/sebastianbergmann/phpunit-selenium.git
+    git clone --depth=1 --branch=1.0.0 git://github.com/sebastianbergmann/phpunit-story.git
+
+    # and the version of phpunit that we expect to run with php 5.2
+    git clone --depth=1 --branch=3.6   git://github.com/sebastianbergmann/phpunit.git
+
+    # fix up the version number of phpunit
+    sed -i 's/@package_version@/3.6-git/g' phpunit/PHPUnit/Runner/Version.php
+
+    # now set up an ini file that adds all of the above to include_path for the PHP5.2 install
+    mkdir -p $HOME/.phpbrew/php/php-5.2.17/var/db
+    echo "include_path=.:${PHP52_PATH}/lib/php:${PHP52_PATH}/lib/php/dbunit:${PHP52_PATH}/lib/php/php-code-coverage:${PHP52_PATH}/lib/php/php-file-iterator:${PHP52_PATH}/lib/php/php-invoker:${PHP52_PATH}/lib/php/php-text-template:${PHP52_PATH}/lib/php/php-timer:${PHP52_PATH}/lib/php/php-token-stream:${PHP52_PATH}/lib/php/phpunit-mock-objects:${PHP52_PATH}/lib/php/phpunit-selenium:${PHP52_PATH}/lib/php/phpunit-story:${PHP52_PATH}/lib/php/phpunit" > ${PHP52_PATH}/var/db/path.ini
+
+    # one more PHPUnit dependency that we need to install using pear under PHP5.2
+    cd $HOME
+    export PHPBREW_RC_ENABLE=1
+    source $HOME/.phpbrew/bashrc
+    phpbrew use 5.2.17
+    pear channel-discover pear.symfony-project.com
+    pear install pear.symfony-project.com/YAML-1.0.2
+
+    # manually go back to the system php, we can't use `phpbrew switch-off`
+    # because we're running a version of php that phpbrew doesn't work with at this point
+    unset PHPBREW_PHP
+    unset PHPBREW_PATH
+    __phpbrew_set_path
+    __phpbrew_reinit
+    eval `$BIN env`
+
+    # clean up build directory
+    rm -rf $HOME/.phpbrew/build/*
+  fi
+
+  # all needed php versions and phpunit versions are installed, either from the above
+  # install script, or from travis cache, so switch to using them
+  cd $HOME
+  export PATH=$HOME/php-utils-bin:$PATH
+  export PHPBREW_RC_ENABLE=1
+  source $HOME/.phpbrew/bashrc
+
+  if [[ ${SWITCH_TO_PHP:0:3} == "5.2" ]]; then
+    phpbrew use 5.2.17
+  else
+    phpbrew use 5.3.29
+  fi
+
+elif [[ ${TRAVIS_PHP_VERSION:0:2} == "5." ]]; then
+  # all other PHP 5.x versions
+  mkdir -p $HOME/phpunit-bin
+  wget -O $HOME/phpunit-bin/phpunit https://phar.phpunit.de/phpunit-4.8.phar
+  chmod +x $HOME/phpunit-bin/phpunit
+  export PATH=$HOME/phpunit-bin/:$PATH
+else
+  composer global require "phpunit/phpunit=5.7.*"
```

Comment:
This should be `composer global require "phpunit/phpunit=6.*"`

See also: https://core.trac.wordpress.org/ticket/41472